pkgs/applications/misc/calibre/default.nix

```diff
@@ -5,25 +5,33 @@
 }:
 
 stdenv.mkDerivation rec {
-  version = "2.64.0";
+  version = "2.66.0";
   name = "calibre-${version}";
 
   src = fetchurl {
     url = "https://download.calibre-ebook.com/${version}/${name}.tar.xz";
-    sha256 = "0jjbkhd3n7rh5q6cl6yy51hyjbxmgm6xj7i2a1d3h2ggrip1zmr9";
+    sha256 = "1dbv6p9cq9zj51zvhfy2b7aic2zqa44lmfmq7k7fkqcgb6wmanic";
   };
 
   inherit python;
 
   patches = [
-    # Patch from Debian that switches the version update change from
-    # enabled by default to disabled by default.
+    # Patches from Debian that:
+    # - disable plugin installation (very insecure)
+    # - disables loading of web bug for privacy
+    # - switches the version update from enabled to disabled by default
+    ./disable_plugins.patch
+    ./links_privacy.patch
```

Comment:
The calibre Debian packaging is managed with bazaar on launchpad here: https://code.launchpad.net/~calibre-packagers/calibre/debian

Here are the links to the patches (fixed revision):
  * https://bazaar.launchpad.net/~calibre-packagers/calibre/debian/download/head:/linksprivacy.patch-20160417214308-6hvive72pc0r4awc-1/links-privacy.patch
  * https://bazaar.launchpad.net/~calibre-packagers/calibre/debian/view/head:/debian/patches/disable_plugins.py